src/main/java/org/mule/tools/apikit/output/MuleConfigGenerator.java

```diff
@@ -68,113 +54,62 @@
                                                                                                        "http://www.mulesoft.org/schema/mule/ee/core"),
                                                                                      "http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd");
 
-  private static final String INDENTATION = "    ";
-
   private final List<GenerationModel> flowEntries;
-  private final File rootDirectory;
-  private final Set<File> ramlsWithExtensionEnabled;
   private final RuntimeEdition runtimeEdition;
-  private final List<API> apis;
-  private final List<ParsingIssue> errors = new LinkedList<>();
+  private final List<ApikitMainFlowContainer> apis;
+  private List<MuleConfig> muleConfigsInApp = new ArrayList<>();
+
+  private final List<ScaffoldingError> scaffoldingErrors = new ArrayList<>();
+  private final List<ScaffolderResource> generatedResources = new ArrayList<>();
```

Comment:
always empty list

Reply:
Yes, the resources will be generated with the munit tests scaffolder.

I left that logic (which is currently empty) because it was defined in the "new scaffolder api" document.

Reply:
remove until required